spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'foods index page' do
+
+  it 'shows sweet potatoes search results' do
+    visit root_path
+
+    fill_in :q, with: 'sweet potatoes'
+
+    click_button 'Search'
+
+    expect(current_path).to eq(foods_path)
+
+    expect(page).to have_content('Total hits: 49676')
+
+    expect(page).to have_content('Item GTIN/UPC code: 8901020020844')
```

Comment:
We are wanting to test that we have a list of 10 foods and all of their attributes not just the upc